AnkiDroid/build.gradle

```
@@ -214,6 +215,8 @@ dependencies {
         resolutionStrategy {
             force 'org.jetbrains:annotations:20.1.0'
         }
+        coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.0.9' // Setting the version of core desugaring library.
+//    coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.0.4'
```

Please only propose code that is useful, no commented out code should make it into PRs, and while comments are usually a good thing they must add information, this one is redundant I think
```suggestion
        coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.0.9'
```